packages/zend-service-rackspace/library/Zend/Service/Rackspace/Servers/Server.php

```diff
@@ -104,11 +104,11 @@ public function __construct($service, $data)
             // require_once 'Zend/Service/Rackspace/Servers/Exception.php';
             throw new Zend_Service_Rackspace_Servers_Exception(self::ERROR_PARAM_CONSTRUCT);
         }
-        if (!array_key_exists('name', $data)) {
+        if (!isset($data['name'])) {
```

Comment:
Note that `array_key_exists` will return `true` of an array value is `NULL` while `isset` will return `false`. 

So this is a functional change....

https://3v4l.org/6llcH

Reply:
You are right @holtkamp ... @Megatherium could you elaborate on this change, please? Did it even make sense to allow `NULL` values in `id` and `name` before? But I tend to agree this should not go like that into a patch release...

Reply:
I have to admit that I was overeager and ignorant of the minute difference.